### PR TITLE
Do not hardcode lib dir

### PR DESCRIPTION
**Pull request by mlehtima**

### CODE REVIEW

**Review by mkosola — APPROVED** [review]

**Comment by martyone** [comment]
There are some peculiarities in this repo I would prefer to follow:

- separate sfos-specific changes (modify rpm/* with separate commit)
- add gerrit's Change-Id header
- add "Sailfish-OS-Specific: yes" to commits that are not meant for upstreaming
- use annotated tags for JB#xxxx references

This allows ch-p'ing commits upstream as-is.

**Review by martyone — APPROVED** [review]
LGTM, please do a fast-forward merge, avoid a true merge commit, thank you.

File: src/lib.pro [hunk 1]
```diff
@@ -18,19 +18,25 @@ public_headers += \
 include(src.pri)
 
 win32: target.path = $${PREFIX}/bin
-else: target.path = $${PREFIX}/lib
+else: target.path = $$[QT_INSTALL_LIBS]
 INSTALLS += target
```
Comment (etaishev) [inline src/lib.pro:22]: @martyone Have you tried building qmllive for the sdk after this?

Now `$$[QT_INSTALL_LIBS]` gets to qmllive-install by relative path, so there is no lib folder

Comment (martyone) [reply src/lib.pro:22]: Indeed, it should have been `$${PREFIX}/$$[QT_INSTALL_LIBS]` I believe. @mlehtima could you please fix that?

Comment (mlehtima) [reply src/lib.pro:22]: $${PREFIX} is already included in $$[QT_INSTALL_LIBS]

Comment (etaishev) [reply src/lib.pro:22]: During qmllive build for the sdk, a custom prefix is set:
https://github.com/sailfishos/sdk-build-tools/blob/master/buildqmllive.sh#L236

Comment (martyone) [reply src/lib.pro:22]: PR opened #8

**Pull request merged**